lib/Helpers/Mail/Content.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * This helper builds the request body for a /mail/send API call.
+ *
+ * PHP version 5.6, 7
+ *
+ * @author    Elmer Thomas <dx@sendgrid.com>
+ * @copyright 2017 SendGrid
+ * @license   https://opensource.org/licenses/MIT The MIT License
+ * @version   GIT: <git_id>
+ * @link      http://packagist.org/packages/sendgrid/sendgrid
+ */
+
+namespace SendGrid\Mail;
+
+class Content implements \JsonSerializable
+{
```

Comment:
You're injecting properties using `constructor`, why are you leaving the `setters`?, it doesn't make any sense, it breaks `OOP encapsulation` and also `immutability`